solano.yml

```diff
@@ -32,6 +32,8 @@ environment:
 
 profiles:
   validate:
+    test_exclude_pattern:
```

Comment:
Does it work to have a test_exclude_pattern at the top level of the config file?  That might be easier to understand.